spec/features/facet.rb

```diff
@@ -2,12 +2,7 @@
 
 describe 'facet', :type => :feature do
 
-  before do
```

Comment:
We should't need to cleanup before and after a test? Assuming we aren't polluting the database from other specs